apisix/plugins/loggly.lua

```diff
@@ -232,13 +233,71 @@ local function send_data_over_udp(message)
 end
 
 
+local function send_bulk_over_http(message, metadata, conf)
+    local endpoint = path.join(metadata.value.host, "bulk", conf.customer_token, "tag", "bulk")
+    local has_prefix = metadata.value.host:find("http")
```

Comment:
Better to use https://github.com/apache/apisix/blob/615ee41312282ca2375abddae1d074637a901d89/apisix/core/string.lua#L46

Reply:
Nice. Thanks for the suggestion.
Updated.